mp/src/game/client/neo/c_neo_player.cpp

```diff
@@ -443,6 +443,8 @@ C_NEO_Player::C_NEO_Player()
 	m_bLastTickInThermOpticCamo = false;
 	m_bIsAllowedToToggleVision = false;
 
+	m_flTocFactor = 0.15f;
+
```

Comment:
For some reason GH won't let me comment on this change:

```diff
- m_pPlayerAnimState = CreatePlayerAnimState(this, CreateAnimStateHelpers(this),
-		NEO_ANIMSTATE_LEGANIM_TYPE, NEO_ANIMSTATE_USES_AIMSEQUENCES);
```

but is this related to this PR? Also, if we remove this initialization, it seems `m_pPlayerAnimState` is no longer referred to anywhere, so should we remove it entirely?

Reply:
Removing INEOPlayerAnimState was part of Tony Sergi's changes/integrations part 2. I must have forgot to remove this particular value, or some weird git merging